dockerplugin.py

```diff
@@ -80,11 +107,13 @@ def emit(container, dimensions, point_type, value, t=None,
     val.meta = {'true': 'true'}
 
     val.values = value
+    log.info('Value to be emitted {value}'.format(value=val))
```

Comment:
If you only have one or two values in your format string, just use `log.info('... {0}'.format(val))`